cognite_toolkit/_cdf_tk/prototypes/featureflag.py

```diff
@@ -0,0 +1,22 @@
+import os
+from functools import lru_cache
+
+import dotenv
+
+
+@lru_cache(maxsize=128)
+def enabled(flag: str) -> bool:
+    """
+    Check if a feature flag is enabled.
+
+    Args:
+        flag (str): The feature flag to check.
+
+    Returns:
+        bool: True if the feature flag is enabled, False otherwise.
+    """
+    dotenv.load_dotenv()
```

Comment:
I don't think this is necessary, in `_cdf.py` in line `187-192` we do load the environment file. This will make the `lru` cache obsolete.